config/zones.json

```diff
@@ -3763,6 +3763,25 @@
     "timezone": "Pacific/Port_Moresby"
   },
   "PH": {
+    "_comment": "Capacity is based on Philippine Department of Energy's Power Situation Report as of 2020 - http://www.doe.gov.ph",
```

Comment:
Can you move this into `DATA_SOURCES.md` with a full link to the report instead?

Reply:
Thank you for considering my input. Included the source in DATA_SOURCES.md